src/interpreter/Environment.ts

```diff
@@ -172,6 +173,26 @@ export class Environment {
                 variableToReturn = this.getMockFunction(lowercaseName);
             }
             return variableToReturn;
+        } else {
+            // Because brs hasn't implemented every built-in global function yet,
+            // allow mocking of functions that "don't exist" in source. This unblocks
+            // unit testing of code that calls the not-yet-implemented global functions.
+            //
+            // TODO: remove this once all built-in global functions are implemented?
+
+            // Don't allow mocks of reserved functions like "init" and "main".
+            if (!name.isReserved) {
+                let mockedFunc = this.getMockFunction(lowercaseName);
+                if (mockedFunc !== BrsInvalid.Instance) {
+                    console.error(
```

Comment:
I went with `console.error` here because importing `Interpreter` caused circular dependency chaos.

Reply:
Boooooo.  That's fair.  We'll eventually get a working solution for that.